### PR TITLE
Fix precision of Usd and Percent to 2 decimal upon serialization

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/model.rs [hunk 1]
```diff
@@ -27,6 +32,7 @@ impl Usd {
         Ok(Usd(result))
     }
 
+    // TODO: Usd * Usd = Usd^2 not Usd !!!
```
Comment (bonomat) [inline daemon/src/model.rs:35]: What do you mean with this todo?

**Review by bonomat — APPROVED** [review]
LGTM 👍

File: daemon/src/model.rs (hunk 1 above)
Comment (thomaseizinger) [reply daemon/src/model.rs:35]: The function signature claims that multiplying `Usd` with `Usd` gives you `Usd` but that is mathematically not correct. The correct result would be `Usd`^2.

You cannot just ignore units when doing arithmetic.

The argument of this function should likely be changed from `Usd` to a scalar like `Decimal` or `u32` or something.

Comment (thomaseizinger) [reply daemon/src/model.rs:35]: @DeliciousHair is going to fix this with #365.

**Review by da-kami — APPROVED** [review]

**Pull request merged**